x-pack/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -71,6 +75,39 @@ export function AlertDetails() {
   const [summaryFields, setSummaryFields] = useState<AlertSummaryField[]>();
   const [alertStatus, setAlertStatus] = useState<AlertStatus>();
 
+  useEffect(() => {
+    if (!alertDetail) {
+      return;
+    }
+
+    const screenDescription = dedent(`The user is looking at an ${
+      alertDetail.formatted.active ? 'active' : 'recovered'
+    } alert.
+    It started at ${new Date(
+      alertDetail.formatted.start
+    ).toISOString()}, and was last updated at ${new Date(
+      alertDetail.formatted.lastUpdated
+    ).toISOString()}.
+
+    ${
+      alertDetail.formatted.reason
+        ? `The reason given for the alert is ${alertDetail.formatted.reason}.`
```

Comment:
i18n?

Reply:
Not needed, everything we sent to the LLM needs to be in English.